src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/UtilityExtension.kt

```diff
@@ -482,6 +513,14 @@ class UtilityExtension : Extension() {
                             return@action
                         }
 
+                        if (threads.get(channel) != null && threads.get(channel)!!.save) {
+                            edit {
+                                content = "**Error:** Only moderators can archive threads that have been set to save."
```

Comment:
This string is confusing. Maybe something like this:

`**Error:** This thread may only be archived by a moderator.`